frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -0,0 +1,59 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
+local StatusWidget = require("ui/widget/statuswidget")
+
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+
+local ReaderStatus = InputContainer:new {
+    document = nil,
+    summary = {
+        rating = 0,
+        note = nil,
+        status = "",
+        modified = "",
+    },
+    enabled = true,
+}
+
+function ReaderStatus:init()
+    if self.ui.document.is_djvu or self.ui.document.is_pdf or self.ui.document.is_pic then
+        self.enabled = false
+        return
+    end
+    UIManager:scheduleIn(0.1, function() self.ui.menu:registerToMainMenu(self) end)
```

Comment:
Because the module ReaderHighlight adds into the menu a Highlight item after the module ReaderStatus. I want to add into the menu the Status item after a Highlight item.